seqr/views/apis/dashboard_api_tests.py

```diff
@@ -120,3 +122,52 @@ def test_export_projects_table(self):
         response = self.client.get('{}?file_format=csv'.format(url))
         self.assertEqual(response.status_code, 400)
         self.assertDictEqual(response.json(), {'error': 'Invalid file_format: csv'})
+
+
+# Tests for AnVIL access disabled
+class LocalDashboardPageTest(AuthenticationTestCase, DashboardPageTest):
+    fixtures = ['users', '1kg_project']
+    NUM_COLLABORATOR_PROJECTS = 2
+    NUM_STAFF_PROJECTS = 3
+    NUM_STAFF_EXPORT_RECS = 3
+    NUM_STAFF_EXPORT_TSV_LINES = NUM_STAFF_EXPORT_RECS + 1
```

Comment:
it is somewhat confusing to define class variables for things that are the same in all the inherited classes. Maybe move the variable definitions into the base class so its really clear whats different, and only leave the variables that are different between them in the subclasses

Reply:
They must be different at least in two of the three classes. Otherwise, we don't have to define them and just use a constant in the test case.